docs/release-process/CHECKLIST.md

```diff
@@ -0,0 +1,84 @@
+## Check list
+
+This section shows what things you need to do before creating a release. This is a manual process as CI does not test everything.
+
+The following example assumes we’re going from version 0.1.0 (`v0.1.0`) to
+0.2.0 (`v0.2.0`).
+
+- Checkout to old release tag
+  - e.g. `git checkout v0.1.0`
+
+- Build `lokoctl` binary from the last release
+  - e.g. `make build`
+  - Copy `lokoctl` binary to your assets directory.
+
+- Deploy `lokomotive` with old release
+  - e.g. `./lokoctl cluster apply`
```

Comment:
Like the previous step says that build using `make build` and current one says `./lokoctl cluster apply`. You don't have lokoctl binary and the lokocfg files in the same place.

We can simply say `make install` in previous step. In in this step we just do `lokoctl cluster apply` from this directory.

Reply:
I think previous step should be changed to use a release binary, so I think it should be OK to build + copy here.